docs/source/_templates/layout.html

```diff
@@ -2,15 +2,22 @@
 
 {% block footer %}
 {{ super() }}
+
+{# Only embed analytics on production #}
+{% if use_analytics %}
 <script>
-  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
-  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
-  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
-  })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
+    (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+    (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+    m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+    })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
 
-      ga('create', 'UA-27761864-11', 'auto');
-      ga('set', 'anonymizeIp', true);
-      ga('send', 'pageview');
+        ga('create', 'UA-27761864-11', 'auto');
+        ga('set', 'anonymizeIp', true);
+        ga('send', 'pageview');
 
 </script>
+<script data-goatcounter="https://docs-conda-io.goatcounter.com/count"
+        async src="_static/js/count.js"></script>
```

Comment:
In Sphinx you should be able to automatically append the JS files via the [`html_js_files`](https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-html_js_files) config option, similar to how pydata theme is [documenting it here](https://pydata-sphinx-theme.readthedocs.io/en/stable/user_guide/static_assets.html). That assumes we have `html_static_path` set correctly.

```python
html_js_files = [
    (
        "js/count.js",
        {
            "data-goatcounter": "https://docs-conda-io.goatcounter.com/count",
            "async": "async",
        },
    ),
]
```